lib/index.js

```diff
@@ -27,11 +27,7 @@ const httpClientHandler = {
       throw new Error('NotFoundError');
```

Comment:
I have to say that this doesn't seem like a great approach to me: If a GET fails for *any reason*, it's turned into a not found error. I'm not really sure what the utility of this `httpClientHandler` is just looking at it -- it seems like an unnecessary abstraction when people could just use `@digitalbazaar/http-client`. From that perspective, this just creates more maintenance work.

Reply:
I think JSON-LD lib downstream depends on that NotFoundError.

Reply:
> I think JSON-LD lib downstream depends on that NotFoundError.

Considering that it's in the message and not in the error name -- I'm skeptical that it's depended on like that. At least, it shouldn't be. If we were going to keep this longer term, the error name should be set to `NotFoundError` -- but it seems like we might just remove it in a future version.